rails-snow-queen/app/assets/stylesheets/application.scss

```diff
@@ -61,4 +61,9 @@ header {
 
 .simple_form abbr {
   text-decoration: none;
+}
+
+#modal_errors {
```

Comment:
Probably better as a class, since you would likely style all your modal errors the same way. In general terms, CSS should use classes. Very rarely will it use IDs

Reply:
but I use that id for Dom manipulation and display in the quote.js and html, that's why i left as an id that the simple form already designates

Reply:
done

Reply:
i am not crazy!!! i must've done it in another dimension ... its REALLY done now